app/graphql/users/resolvers.js

```diff
@@ -0,0 +1,35 @@
+const { user: User } = require('../../models'),
+  { userLoggedIn } = require('../events');
+
+const getUser = (_, params) => User.getOne(params);
+const getUsers = (_, params) => User.getAll(params);
+
+const createUser = (_, { user }) => User.createModel(user);
+const logIn = (_, { credentials }) => {
+  // IMPORTANT: Not a functional login, its just for illustrative purposes
+  userLoggedIn.publish(credentials.username);
+  return {
+    accessToken: 'example_token',
+    refreshToken: 'example_refresh_token',
+    expiresIn: 134567899123
```

Comment:
I think that 1234567899123 doesn't fit in `GraphQLInt`.  `logIn` would throw an error (correct me if I'm wrong). I know it's not a functional login but maybe it's better if no error is thrown so every query/mutation can be tested at the beginning of the training/project.

Reply:
Got it, I will look into it